framework/widgets/ActiveField.php

```diff
@@ -315,10 +317,17 @@ public function error($options = [])
      *
      * @return $this the field object itself
      */
-    public function hint($content, $options = [])
+    public function hint($content = null, $options = [])
```

Comment:
I think I'll merge it w/o signature change in 2.0 and w/ signature change in 2.1.